src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -225,47 +225,50 @@ export class FindAndReplacePlugin extends UIPlugin {
   // ---------------------------------------------------------------------------
   // Replace
   // ---------------------------------------------------------------------------
-  /**
-   * Replace the value of the currently selected match
-   */
-  private replace(replaceWith: string) {
-    if (this.selectedMatchIndex === null || !this.currentSearchRegex) {
+  private replaceMatch(replaceWith: string, selectedMatch: SearchMatch) {
```

Comment:
I'd rather have the arguments swapped
```suggestion
  private replaceMatch(selectedMatch: SearchMatch, replaceWith: string, ) {
```